src/pymodaq_plugins_keithley/daq_viewer_plugins/plugins_0D/daq_0Dviewer_Keithley27XX.py

```diff
@@ -0,0 +1,248 @@
+            {'title': 'Resources', 'name': 'resources', 'type': 'list', 'limits': resources_list,
+             'value': resources_list[0]},
+            {'title': 'Keithley', 'name': 'Keithley_Params', 'type': 'group', 'children': [
+                {'title': 'ID', 'name': 'ID', 'type': 'text', 'value': ''},
+                {'title': 'FRONT panel', 'name': 'frontpanel', 'type': 'group', 'children': [
+                    {'title': 'Mode', 'name': 'frontmode', 'type': 'list',
+                     'limits': ['VOLT:DC', 'VOLT:AC', 'CURR:DC', 'CURR:AC', 'RES', 'FRES', 'FREQ', 'TEMP'],
+                     'value': 'VOLT:DC'}]}
+            ]}
+        ]
+    elif panel == 'REAR':
+        params = comon_parameters + [
+            {'title': 'Resources', 'name': 'resources', 'type': 'list', 'limits': resources_list,
+             'value': resources_list[0]},
+            {'title': 'Keithley', 'name': 'Keithley_Params', 'type': 'group', 'children': [
+                {'title': 'ID', 'name': 'ID', 'type': 'text', 'value': ''},
+                {'title': 'REAR panel', 'name': 'rearpanel', 'type': 'group', 'children': [
+                    {'title': 'Mode', 'name': 'rearmode', 'type': 'list',
+                     'limits': ['SCAN_LIST', 'VOLT:DC', 'VOLT:AC', 'CURR:DC', 'CURR:AC', 'RES', 'FRES', 'FREQ', 'TEMP'],
+                     'value': 'SCAN_LIST'}
+                ]}
+            ]}
+        ]
+
+    def __init__(self, parent=None, params_state=None):
+        super(DAQ_0DViewer_Keithley27XX, self).__init__(parent, params_state)
+
+    def ini_attributes(self):
+        """Attributes init when DAQ_0DViewer_Keithley class is instancied"""
+        print("Panel configuration 0D_viewer: ", self.panel)
+        self.channels_in_selected_mode = None
+
+    def commit_settings(self, param: Parameter):
+        """Apply the consequences of a change of value in the detector settings
+
+        :param Parameter: A given parameter (within detector_settings) whose value has been changed by the user
+        """
+        if 'mode' in param.name():
+            """Updates the newly selected measurement mode"""
+            # Read the configuration file to determine which mode to use and send corresponding instruction to driver
+            if self.panel == 'FRONT':
+                value = param.value()
+                self.controller.set_mode(value)
+            elif self.panel == 'REAR':
+                value = 'SCAN_' + param.value()
+                self.channels_in_selected_mode = self.controller.set_mode(value)
+
+    def ini_detector(self, controller=None):
+        """Detector communication initialization
+
+        :param controller: Custom object of a PyMoDAQ plugin (Slave case). None if only one actuator/detector by controller (Master case)
+        :type controller: object
+
+        :return: Initialization status, false if it failed otherwise True
+        :rtype: bool
+        """
+        print('Detector 0D initialized')
```

Comment:
do not use print but use logger
you can instanciate a logger instance in your module with

```
from pymodaq.utils.logger import set_logger, get_module_name
logger = set_logger(get_module_name(__file__))
```

then call one of its method:

logger.info, logger.warning ...

or print quick messages in the gui using:
```
self.emit_status(ThreadCommand('Update_Status', ['Message to print']))
```

Reply:
All print deleted or replaced by logger.info / warning or error